tests/source-sftp/setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+export SFTP_ADDRESS="${SFTP_ADDRESS:=source-sftp-db-1.flow-test:2222}"
+export SFTP_USERNAME="${SFTP_USERNAME:=test}"
+export SFTP_PASSWORD="${SFTP_PASSWORD:=test}"
+export TEST_STREAM="${TEST_STREAM:=/tests}"
+
+# Set ID_TYPE to string because parsing CSV files will always result in string values.
+export ID_TYPE=string
+
+export RESOURCE="{ \"stream\": \"${TEST_STREAM}\", \"syncMode\": \"incremental\" }"
+
+docker compose -f source-sftp/docker-compose.yaml up --detach
+
+# Wait for the sftp server to be ready.
+go run tests/source-sftp/wait_for_server.go
```

Comment:
Nice use of a helper program 👍

Should the address/username/password flags maybe be specified explicitly? (Unless you're deliberately taking advantage of having the address be `source-sftp-db-1.flow-test:2222` for the capture connector within Docker but `localhost:2222` when running this binary, I guess?)

Reply:
The defaults do happen to work for the instance set up by the automated tests, but I think having the flags set is more clear so I'll add those.